troposphere/static/js/components/images/list/common/ImageListCard.react.js

```diff
@@ -42,7 +42,7 @@ export default React.createClass({
             name = image.get('name');
 
         if (imageTags.length > 10) {
-            imageTags = imageTags.slice(0,10);
+            imageTags = imageTags.first(10);
```

Comment:
This entire if block can be replaced by:
`imageTags = imageTags.first(10)`
I would add a comment above.